crates/uv-distribution/src/source/mod.rs

```diff
@@ -217,6 +217,7 @@ impl<'a, T: BuildContext> SourceDistributionBuilder<'a, T> {
                         let url = Url::from_file_path(path).expect("path is absolute");
 
                         // If necessary, extract the archive.
+                        // I don't think this is allowed for --find-links?
                         let extracted = extract_archive(path, self.build_context.cache()).await?;
```

Comment:
Should this be addressed?

Reply:
Lol no sorry. Stray comment from a separate refactor.

Reply:
Thanks for catching.